packages/mantine/src/components/code-editor/CodeEditor.tsx

```diff
@@ -143,6 +141,10 @@ export const CodeEditor: FunctionComponent<CodeEditorProps> = (props) => {
         }
     };
 
+    const [hasMonacoError, setHasMonacoError] = useState(false);
+    const renderErrorOutline = useMemo(() => !!error || hasMonacoError, [hasMonacoError, error]);
+    const {classes, theme} = useStyles({error: renderErrorOutline});
```

Comment:
Could you add some tests?

Reply:
added an assert in existing test to verify the border. It's really hard to test the case of a marker coming from `monaco` itself, since we mock the entire monaco editor.